logsmanagement/README.md

```diff
@@ -0,0 +1,676 @@
+| `log path` | The path to the web server's `access.log`. If set to `auto`, the collector will attempt to auto-discover it, provided the name of the configuration section is either `Apache access.log` or `Nginx access.log`. |
+| `use inotify` | Select between inotify and file stat watchers (providing `libfluent-bit.so` has been built with inotify support). It defaults to `yes`. Set to `no` if abnormally high CPU usage is observed or if the log source is expected to consistently produce tens of thousands of (unbuffered) logs per second. |
+| `log format` | The log format to be used for parsing. Unlike the [`GO weblog`]() module, only the `CSV` parser is supported and it can be configured [in the same way](https://github.com/netdata/go.d.plugin/blob/master/modules/weblog/README.md#known-fields) as in the `GO` module. If set to `auto`, the collector will attempt to auto-detect the log format using the same logic explained [here](https://github.com/netdata/go.d.plugin/blob/master/modules/weblog/README.md#log-parser-auto-detection). |
+| `verify parsed logs` | If set to `yes`, the parser will attempt to verify that the parsed fields are valid, before extracting metrics from them. If they are invalid (for example, the response code is less than `100`), the `invalid` dimension will be incremented instead. Setting this to `no` will result in a slight performance gain. |
+| `vhosts chart` | Enable chart showing names of the virtual hosts extracted from the collected logs. |
+| `ports chart` | Enable chart showing port numbers extracted from the collected logs. |
+| `IP versions chart` | Enable chart showing IP versions (`v4` or `v6`) extracted from the collected logs. |
+| `unique client IPs - current poll chart` | Enable chart showing unique client IPs in each collection interval. |
+| `unique client IPs - all-time chart` | Enable chart showing unique client IPs since agent startup. It is recommended to set this to `no` as it can have a negative impact on long-term performance. |
+| `http request methods chart` | Enable chart showing HTTP request methods extracted from the collected logs. |
+| `http protocol versions chart` | Enable chart showing HTTP protocol versions exctracted from the collected logs. |
+| `bandwidth chart` | Enable chart showing request and response bandwidth extracted from the collected logs. |
+| `timings chart` | Enable chart showing request processing time stats extracted from the collected logs. |
+| `response code families chart` | Enable chart showing response code families (`1xx`, `2xx` etc.) extracted from the collected logs. |
+| `response codes chart` | Enable chart showing response codes extracted from the collected logs. |
+| `response code types chart` | Enable chart showing response code types (`success`, `redirect` etc.) extracted from the collected logs. |
+| `SSL protocols chart` | Enable chart showing SSL protocols (`TLSV1`, `TLSV1.1` etc.) exctracted from the collected logs. |
```

Comment:
Do we only support `TLS` protocols? If this is not the case, and we still support `SSL 3`, please, give additional information here. If it has relationship with web server, it is also good to add this information.

Reply:
I believe it's better as is, because we won't have to remember to update the documentation if more protocols are added in the future.